docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,462 @@
+
+- **Using a [Dynamic Function](#dynamic-functions)**: This will dynamically render components at request time, but may reuse cached data, if any.
+- **Opting out of the [Data Cache](#data-cache)**: This will fetch data on every request, which in turn will skip the Render Cache.
+- **Using the route segment config options `dynamic = force-dynamic` or `revalidate = 0`**: This will skip the Render Cache and the Data Cache. Meaning components will be rendered and data fetched on every request.
+
+> **Static vs. Dynamic Rendering**:
+>
+> You may see the terms **Static Rendering** and **Dynamic Rendering** being used. Static Rendering refers to caching the render output on the server, at build time, whereas Dynamic Rendering refers to rendering components on every request, at runtime.
+
+### Router Cache
+
+Next.js has an in-memory client-side cache that stores the React Server Component Payload, split by individual routes. This **Router Cache** is used to improve the navigation experience. Next.js [prefetches](/docs/app/building-your-application/routing/linking-and-navigating#prefetching) and caches routes that the user is likely going to navigate to (when using the `<Link >` component), and caches visited route segments, enabling [partial rendering](/docs/app/building-your-application/routing/linking-and-navigating#partial-rendering). This applies to newly visited routes, as well as backward/forward browser navigation.
+
+For the user, this results in [soft navigation](/docs/app/building-your-application/routing/linking-and-navigating#soft-navigation) (no page reloads) and faster page transitions; it also reduces unnecessary requests to the server.
+
+> **Good to know:**
+>
+> You may see the Router Cache being referred to as **Client-side Cache** or **Prefetch Cache**. While **Prefetch Cache** refers to the prefetched route segments, **Client-side Cache** refers to the whole Router cache, which includes both visited and prefetched segments.
+> This cache specifically applies to Next.js and Server Components, and is different to the browser's [bfcache](https://web.dev/bfcache/).
+
+#### Duration
+
+The cache is stored in the browser's temporary memory. Two factors determine how long the router cache lasts:
+
+- **Session**: The cache persists across [soft navigations](/docs/app/building-your-application/routing/linking-and-navigating#soft-navigation). However, it's cleared on page refresh or [hard navigation](/docs/app/building-your-application/routing/linking-and-navigating#hard-navigation).
+- **Automatic invalidation period**: The cache of an individual segment is automatically invalidated after a specific time. The duration depends on whether the route is [statically rendered](/docs/app/building-your-application/rendering/static-and-dynamic-rendering#static-rendering-default) or [dynamically rendered](/docs/app/building-your-application/rendering/static-and-dynamic-rendering#dynamic-rendering):
+  - **Dynamic routes**: 30 seconds
```

Comment:
```suggestion
  - **Dynamically rendered**: 30 seconds
```

Reply:
Nice! This is what I meant in [my comment](https://github.com/vercel/next.js/pull/52514#discussion_r1262443786).